### PR TITLE
Adding README

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+They are looking to see if you have properly coded the `Message` class so that the tests pass.
+They will then turn their attention to the `Rover` class. Here, you need to complete the tests for the `Rover` class and add some functionality to the class itself. 
+
+After they review your code, they will run the code as-is to verify that you have completed all the tests and that all the tests pass.
+You should have 13 tests that pass. If you have completed the bonus mission, you should have 14 tests that pass.
+
+### Grading Tests
+
+After these initial checks, your TA will run the tests inside `studentgrade.spec.js`. You may have noticed this file already, but this file only contains grading tests for your TA to use.
+If you would like to see if your work will pass the grading tests, you can insert it into the `spec/` folder and then run the tests.
```
Comment (codinglikeagirl42) [inline README.md]: These directions are confusing as `studentgrade.spec.js` is already in the `spec/` folder in the starter code.

**Review by codinglikeagirl42 — CHANGES_REQUESTED** [review]
This was a brief review of the readme with anticipation of changes in Q1. At which time, the readme will need to be resubmitted for a formal QA review.

File: README.md (hunk 1 above)
Comment (gildedgardenia) [reply README.md]: Done!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+First, your TA is going to check your code.
+They are looking to see if you have properly coded the `Message` class so that the tests pass.
+They will then turn their attention to the `Rover` class. Here, you need to complete the tests for the `Rover` class and add some functionality to the class itself. 
+
+After they review your code, they will run the code as-is to verify that you have completed all the tests and that all the tests pass.
+You should have 13 tests that pass. If you have completed the bonus mission, you should have 14 tests that pass.
+
+### Grading Tests
+
+After these initial checks, your TA will run the tests inside `studentgrade.spec.js`. You may have noticed this file already, but this file only contains grading tests for your TA to use.
```
Comment (codinglikeagirl42) [inline README.md]: If this file is already in the spec folder, aren't these tests already being run?  Do we need to remove this file from the folder to begin with? 

Instructions for TA's from gitlab -> https://gitlab.com/LaunchCodeEducation/github-classroom-autograded-assignments/js-unit-1/mars-rover

Comment (gildedgardenia) [reply README.md]: I added more clarification for students on how to insert it. The TAs would know this as they are industry professionals.

**Review by codinglikeagirl42 — APPROVED** [review]
Line 23 and 24 are speaking to the student and TA running tests, it is unclear how to run these tests.  There is no policy in place that calls out how this should be done… Passing for now.

**Pull request merged**